src/test/java/io/confluent/connect/elasticsearch/integration/BaseConnectorIT.java

```diff
@@ -89,4 +91,29 @@ protected long waitForConnectorToStart(String name, int numTasks) throws Interru
       return Optional.empty();
     }
   }
+
+  public long waitForConnectorToFail(String name, int numTasks, String trace) throws InterruptedException {
+    TestUtils.waitForCondition(
+        () -> assertConnectorAndTasksFailed(name, numTasks, trace).orElse(false),
+        CONNECTOR_STARTUP_DURATION_MS,
+        "Connector tasks did not fail in time."
+    );
+    return System.currentTimeMillis();
```

Comment:
Nit: we don't seem to be using this returned value (nor we use the return value in `waitForConnectorToStart`, which seems to be what this method was copied from).

Reply:
Sure, deleted.